content/docs/command-reference/import-url.md

```diff
@@ -140,6 +141,11 @@ produces a regular stage in `dvc.yaml`.
   finish the operation(s)); or if the target data already exist locally and you
   want to "DVCfy" this state of the project (see also `dvc commit`).
```

Comment:
Also instead of "download everything later" it should be "import the data later" now, I think. Since there's a separate `--no-download` option now (so let's only use term _download_ there).

Reply:
I would get rid of the second part ("DVCfy..."). It's not clear what it's supposed to do, and the command will fail if a file with the same of what is being imported already exists locally.